week1/task4/Task4.cs

```diff
@@ -0,0 +1,20 @@
+int n = 6;
+int nthNumber = Fibonacci(n);
+Console.WriteLine($"Nth fibonacci number is {nthNumber}");
+
+int Fibonacci(int n)
+{
+    List<int> fibonacciArray = new List<int>();
```

Comment:
This looks good, could you solve this with only using variables without using List?